plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -530,7 +538,7 @@ private void indexRemoteModel(
                     String modelId = modelMetaRes.getId();
                     mlTask.setModelId(modelId);
                     log.info("create new model meta doc {} for upload task {}", modelId, taskId);
-                    mlTaskManager.updateMLTask(taskId, ImmutableMap.of(MODEL_ID_FIELD, modelId, STATE_FIELD, COMPLETED), 5000, true);
+                    mlTaskManager.updateMLTask(taskId, Map.of(MODEL_ID_FIELD, modelId, STATE_FIELD, COMPLETED), 5000, true);
```

Comment:
Why change to Map.of()? This also returns an immutable Map so nothing changed?

Reply:
It's by PE's recommendation.